modules/rhcos-about.adoc

```diff
@@ -77,6 +77,10 @@ extracted, and written to disk, then the bootloader is modified to boot into
 the new version. The machine will reboot into the update in a rolling manner to
 ensure cluster capacity is minimally impacted.
 
+* **bootupd firmware and bootloader updater**: Package managers and hybrid systems such as rpm-ostree do not update the firmware or the bootloader. With bootupd, {op-system} users have access to a cross-distribution, system-agnostic OS update tool that manages firmware and boot updates in UEFI and legacy BIOS boot modes that run on modern architectures such as ppc64le and aarch64.
```

Comment:
This works for me but I modified it slightly for some nits :-)

```suggestion
* **bootupd firmware and bootloader updater**: Package managers and hybrid systems such as rpm-ostree do not update the firmware or the bootloader. With bootupd, {op-system} users have access to a cross-distribution, system-agnostic update tool that manages firmware and boot updates in UEFI and legacy BIOS boot modes that run on modern architectures such as x86_64, ppc64le, and aarch64.
```

Reply:
Excellent, thanks @ashcrow - I have updated with your suggestions. Could you please ack if this looks good to you and I will merge?

Reply:
Looks good!